install/README.md

```diff
@@ -171,7 +171,7 @@ communicates with at once, or specific devices can be targeted for a firmware up
 
 > **IMPORTANT:**
 > Before FAS can be used to update firmware, refer to the 1.5 _HPE Cray EX System Software Getting Started Guide S-8000_
```

Comment:
Should "1.5" be included here? For Papaya, the guide is version 22.11 or edition 5. Is this a reference to an older version of the guide that should be removed?

```suggestion
> Before FAS can be used to update firmware, refer to the _HPE Cray EX System Software Getting Started Guide S-8000_
```